weathervane.pl

```diff
@@ -128,6 +128,10 @@ sub getAppInstanceHostOrCluster {
 	}
 	
 	if ($appInstanceHostname) {
+		if ( ($appInstanceHostname eq "localhost") || ($appInstanceHostname eq "127.0.0.1") ) {
+		  $console_logger->error("Hostname $appInstanceHostname was specified for appInstanceHost, but is not valid.");
```

Comment:
Rather than saying "is not valid", you might be explicit and say something like "localhost and associated IP addresses cannot be used in hostnames"

Reply:
How about:
Hostname $appInstanceHostname was specified for appInstanceHost, but a remotely accessible address must be used.

Reply:
Sounds good to me.